dev/tests/api-functional/testsuite/Magento/GraphQl/Sales/RetrieveOrdersByOrderNumberTest.php

```diff
@@ -404,6 +404,52 @@ public function testGetMatchingOrdersForLowerQueryLength()
         $this->assertCount($response['customer']['orders']['total_count'], $response['customer']['orders']['items']);
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/GraphQl/Sales/_files/orders_with_customer.php
+     * @return void
+     * @throws AuthenticationException
+     */
+    public function testGetCustomerOrdersWithChangedDirect()
```

Comment:
What do you think about renaming it to the following?
```suggestion
    public function testGetCustomerDescendingSortedOrders()
```